metrics/bigfiles_high_water_mark

```diff
@@ -1 +1 @@
-1094
+1146
```

Comment:
`records_mover/db/postgres/copy_options/date_input_style.py` got bigger.  This PR sets up enough testing for a future PR to shrink it by moving case logic into something more general.

```sh
389: tests/integration/*****
381: records_mover/db/postgres/copy_options/date_input_style.py
376: setup.py

Reduce total number of bigfiles violations to 1143 or below!

Tasks: TOP => quality
(See full trace by running task with --trace)
Existing violations: 1143
Found 1146 bigfiles violations
```